src/terminal.ts

```diff
@@ -447,6 +447,21 @@ export class FlatpakManifest {
       await config.update(configName, commandPath)
     }
   }
+
+  async deleteOverridenWorkspaceConfig(
+    section: string,
+    configName: string,
+    command: string
+  ): Promise<void> {
+    const commandPath = path.join(this.buildDir, `${command}.sh`)
+    await this.runInRepo(command, true).save(commandPath)
```

Comment:
why you need to save the command file? i suppose you want to remove it if it exists?

Reply:
that line was an oversight.. What is the preferred action here, delete the file or just leave it alone?

Reply:
I would say, leave it alone; it's supposed to be a directory that's part of your gitignore anyway